src/components/learner-credit-management/data/constants.js

```diff
@@ -111,6 +111,14 @@ export const START_DATE_DEFAULT_TO_TODAY_THRESHOLD_DAYS = 14;
 // Default empty content_price value
 export const EMPTY_CONTENT_PRICE_VALUE = 0;
 
+// Course Run availability map
+export const COURSE_AVAILABILITY_MAP = {
```

Comment:
[nit/clarification] Could we re-use the existing `COURSE_AVAILABILITY_MAP` defined [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/f1887e42b7fc39623b56184c17258ef9a3f578fa/src/components/app/data/constants.js#L21-L26)?

Reply:
Moot point since I ended up removing it for the `isLateEnrollmentEligible`